tsconfig.test.json

```diff
@@ -10,6 +10,7 @@
 		]
 	},
 	"include": [
-		"**/*.test.ts"
+		"**/*.test.ts",
+		"__mocks__/**/*.ts"
```

Comment:
Note: was needed to silence the linter about `@actions/github.ts` file in `__mocks__` folder